### PR TITLE
[Docs-2.0] Update `ContributionChecklist.md`

### PR DESCRIPTION
**Pull request by BTCparadigm**

### CODE REVIEW

**Comment by yahiheb** [comment]
@BTCparadigm You need to fix the conflicts

File: docs/building-wasabi/ContributionChecklist.md [hunk 1]
```diff
@@ -10,17 +10,22 @@
 So you're interested in contributing to Wasabi - welcome!
 This checklist will get you plugged in and productive as quickly as possible.
 
+:::tip Embrace the open-source cryptoanarchy!
```
Comment (MarnixCroes) [inline docs/building-wasabi/ContributionChecklist.md:13]:
```suggestion
:::tip Embrace the open-source crypto-anarchy!
```

File: docs/building-wasabi/ContributionChecklist.md [hunk 2]
```diff
@@ -10,17 +10,22 @@
...
+It is not required to work on an existing issue to contribute to Wasabi, and no one is here to tell you what to do.
+Contributors who have their own ideas are free to work in their own forks on whatever they wish, however they wish, and without any permission from anyone.
```
Comment (MarnixCroes) [inline docs/building-wasabi/ContributionChecklist.md:15]:
```suggestion
Contributors who have their own ideas are free to work at their own forked version on whatever they wish, however they wish and without any permission from anyone.
```

File: docs/building-wasabi/ContributionChecklist.md [hunk 3]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
...
-Ok. You’re all set up and ready to work. Here’s what to do next.
-1. **Find a problem somewhere in Wasabi-land** that (a) needs fixing and (b) is a match for your skills and interests.
+## What to work on
+Education and development are vitally important aspects of Bitcoin privacy and Wasabi Wallet specifically.
+The more Wasabi users there are, the better their privacy, because [anonymity loves company](https://www.freehaven.net/anonbib/cache/usability:weis2006.pdf).
+The better UX and privacy Wasabi provides, the more there will be users, because of [network effect](https://en.wikipedia.org/wiki/Network_effect).
+
+### Education Contributor
+1. **Videos, podcasts, memes and articles** are great ways of sharing information to your peers.
+You can create wallet tutorial videos, insightful recorded discussions, articles or memes about Wasabi and privacy.
```
Comment (MarnixCroes) [inline docs/building-wasabi/ContributionChecklist.md:58]:
```suggestion
You can create wallet tutorials, record insightful discussions, write articles or make memes about Wasabi and privacy.
```

File: docs/building-wasabi/ContributionChecklist.md [hunk 4]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
...
+2. **Organizing events and doing presentations** about Wasabi Wallet and Bitcoin privacy.
+Connecting like-minded individuals, who want to take control of their own financial privacy and teaching them about the awesome tools that are available, can be life changing.
+3. **Helping and teaching new users**, once you are familiar with the software, is something that can't be underestimated.
```
Comment (MarnixCroes) [inline docs/building-wasabi/ContributionChecklist.md:61]:
```suggestion
3. **Helping and teaching new users**, being familiar with the software is something that can't be underestimated.
```

File: docs/building-wasabi/ContributionChecklist.md [hunk 5]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
...
+You can do this in any social media like Twitter, Telegram, Reddit etc or in your own neighborhood.
+
+:::tip [zkSNACKs Ltd.](https://zksnacks.com/) is known for compensating some of the educators when they consistently help out new users or create good content.
```
Comment (MarnixCroes) [inline docs/building-wasabi/ContributionChecklist.md:64]:
```suggestion
:::tip [zkSNACKs Ltd.](https://zksnacks.com/) is known for compensating educators who consistently help out new users or create good content. 
```

**Review by MarnixCroes — CHANGES_REQUESTED** [review]
Small improvements

File: docs/building-wasabi/ContributionChecklist.md [hunk 6]
```diff
@@ -10,17 +10,22 @@
 So you're interested in contributing to Wasabi - welcome!
 This checklist will get you plugged in and productive as quickly as possible.
 
+:::tip Embrace the open-source crypto-anarchy!
+It is not required to work on an existing issue to contribute to Wasabi, and no one is here to tell you what to do.
+Contributors who have their own ideas are free to work at their own forked version on whatever they want, however they wish and without any permission from anyone.
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionChecklist.md:15]: How about this:

```suggestion
No one is here to tell you what to do, and it is not required to work on an existing issue to contribute to Wasabi.
Contributors are free to work on their own forked versions on whatever they want, however they wish and without any permission from anyone.
```

File: docs/building-wasabi/ContributionChecklist.md [hunk 7]
```diff
@@ -10,17 +10,22 @@
...
+With that being said, in order to maximize the usefulness/impact of your work, it's a good idea to consult with other Wasabikas via Slack, GitHub, or other communication channels before setting out on any serious contribution effort.
+:::
+
 [[toc]]
 
 ---
 
 ## Who is a contributor?
-
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionChecklist.md:18]: Why removing these empty lines?
We have them in all documents so we better be consistent and keep them after the titles.

File: docs/building-wasabi/ContributionChecklist.md [hunk 8]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
 This will help others point you in the right direction.
 - Explore the communication channels and find out what the peers are tinkering on, learn about the project and who is contributing in what way.
 This will help you to find the interesting challenges you can work on.
-- Follow [@WasabiWallet](https://twitter.com/wasabiwallet) on Twitter and subscribe to the [Wasabi YouTube channel](https://www.youtube.com/channel/UCobsrSexTuVkL39mbrQ35VQ) to stay up-to-date.
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionChecklist.md:33]: Why this got removed?

File: docs/building-wasabi/ContributionChecklist.md [hunk 9]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
...
 Everyone is welcome to join the discussions. 
 However, being familiar with the topic (if there is any) is recommended.
 Discussions might be recorded and published on the Wasabi Wallet [YouTube channel](https://www.youtube.com/c/wasabiwallet).
 You can find the schedule, call link, past recordings and more at the [Wasabi Research Club repository](https://github.com/zkSNACKs/WasabiResearchClub/).
 
-## Learn how we work
-- To understand how Wasabi CoinJoins work, read [Zero Link: The Bitcoin Fungibility Framework](https://github.com/nopara73/zerolink) and explore the [Documentation](/using-wasabi/CoinJoin.md).
+## Learn how Wasabikas work
+- Educate yourself about the software by reading through [Using Wasabi](/using-wasabi/).
+- Learn what Wasabikas have planned for the [future Of Wasabi Wallet](/building-wasabi/FutureOfWasabiWallet.md).
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionChecklist.md:46]: At this point there is no such document.

File: docs/building-wasabi/ContributionChecklist.md [hunk 10]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
...
 - Familiarize yourself with [C4: The Collective Code Construction Contract](https://rfc.unprotocols.org/spec:1/C4/).
 It’s a simple set of collaboration rules based on GitHub’s fork and pull request model, and a foundational part of how we work together.
 
-## Do valuable work
-Ok. You’re all set up and ready to work. Here’s what to do next.
-1. **Find a problem somewhere in Wasabi-land** that (a) needs fixing and (b) is a match for your skills and interests.
+## What to work on
+Education and development are vitally important aspects of Bitcoin privacy and Wasabi Wallet specifically.
+The more Wasabi users there are, the better their privacy, because [anonymity loves company](https://www.freehaven.net/anonbib/cache/usability:weis2006.pdf).
+The better UX and privacy Wasabi provides, the more there will be users, because of [network effect](https://en.wikipedia.org/wiki/Network_effect).
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionChecklist.md:54]: Do you want these to be on separate lines? If yes then you need to separate them by an empty line or better to use `-` at the start of each line.

File: docs/building-wasabi/ContributionChecklist.md [hunk 11]
```diff
@@ -30,21 +35,39 @@ All such contributions are very welcomed and greatly appreciated.
...
+You can create wallet tutorials, record insightful discussions, write articles or make memes about Wasabi and privacy.
+2. **Organizing events and doing presentations** about Wasabi Wallet and Bitcoin privacy.
+Connecting like-minded individuals, who want to take control of their own financial privacy and teaching them about the awesome tools that are available, can be life changing.
+3. **Helping and teaching new users**, once you are familiar with the software, is something that can't be underestimated.
+You can do this in any social media like Twitter, Telegram, Reddit etc or in your own neighborhood.
+
+:::tip [zkSNACKs Ltd.](https://zksnacks.com/) is known for compensating educators who consistently help out new users or create good content. 
+The company is actively sponsoring Bitcoin events and organizing [Contribution Games](https://blog.wasabiwallet.io/coming-soon-contribution-games/).
+In case you have been contributing already, feel free to let other Wasabikas know by sending us a link in [Slack](https://join.slack.com/t/tumblebit/shared_invite/enQtNjQ1MTQ2NzQ1ODI0LWIzOTg5YTM3YmNkOTg1NjZmZTQ3NmM1OTAzYmQyYzk1M2M0MTdlZDk2OTQwNzFiNTg1ZmExNzM0NjgzY2M0Yzg).
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionChecklist.md:66]: Which link?

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/building-wasabi/ContributionChecklist.md (hunk 7 above)
Comment (BTCparadigm) [reply docs/building-wasabi/ContributionChecklist.md:18]: Consistency was what I was aiming for. I'll add a space then

File: docs/building-wasabi/ContributionChecklist.md (hunk 8 above)
Comment (BTCparadigm) [reply docs/building-wasabi/ContributionChecklist.md:33]: Didnt see following Wasabi Twitter or Youtube as necessary for new contributors.

File: docs/building-wasabi/ContributionChecklist.md (hunk 9 above)
Comment (BTCparadigm) [reply docs/building-wasabi/ContributionChecklist.md:46]: Can we leave it as a placeholder?
It was there to remind me.

File: docs/building-wasabi/ContributionChecklist.md (hunk 11 above)
Comment (BTCparadigm) [reply docs/building-wasabi/ContributionChecklist.md:66]: Link to their content but I guess that can be explained a bit better.

**Review by yahiheb — APPROVED** [review]
LGTM

**Pull request merged**